### ISSUE TITLE
Keyword searching from collection landing pages not working correctly

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/dams_project_mgmt#459]
### Descriptive Summary

Kate reported this issue via LibAnswers as follows:  "When I navigate to a specific collection (such as Santa Cruz County Historical Photographs), a keyword search does not produce any results (or rather, the results list is the entire collection). But when I search a collection group (such as Santa Cruz County History Collections), the same keyword search produces results."

For example, From the landing page for the Santa Cruz County History Collections (https://digitalcollections.library.ucsc.edu/collections/sj139538x), I searched for 'surfing' and the system returned 8 results.  

<img width="883" alt="Screen Shot 2021-04-22 at 9 09 23 AM" src="https://user-images.githubusercontent.com/32003407/115772131-79204f00-a34a-11eb-894e-c15613751517.png">

<img width="775" alt="Screen Shot 2021-04-22 at 8 39 36 AM" src="https://user-images.githubusercontent.com/32003407/115772163-7faec680-a34a-11eb-8fd7-23f13b565a4a.png">

However, when attempting the same keyword search, 'surfing', on the landing pages for any of the member collections, e.g. Vester Dick (https://digitalcollections.library.ucsc.edu/collections/p5547v91x), the system returns a result list inclusive of everything in that member collection. 

<img width="506" alt="Screen Shot 2021-04-22 at 9 10 32 AM" src="https://user-images.githubusercontent.com/32003407/115772334-aff66500-a34a-11eb-9591-8eedd2469850.png">
<img width="745" alt="Screen Shot 2021-04-22 at 8 40 04 AM" src="https://user-images.githubusercontent.com/32003407/115772340-b2f15580-a34a-11eb-9442-b4036cc4779a.png">

I saw the same pattern when keyword searching 'commencement' in the Campus history super-collection and then within the Eric Thiermann member collection.

### Background 

I'm wondering if this has something to do with how that super-collections/collection groups were set up in the system versus the collections.  Despite both layers technically being collections, there are some disparities between the two as we've already seen with the metadata editing form.  

Or it could be an issue with keyword searching using the search box that's part of the facet panel, because the same issue occurs with Harry Mayo which is not part of a super-collection, but has a collection landing page layout as the collections that are part of super-collections.

<img width="527" alt="Screen Shot 2021-04-22 at 9 15 28 AM" src="https://user-images.githubusercontent.com/32003407/115772914-62c6c300-a34b-11eb-808d-d7c24b7b8f60.png">
<img width="593" alt="Screen Shot 2021-04-22 at 9 15 42 AM" src="https://user-images.githubusercontent.com/32003407/115772922-63f7f000-a34b-11eb-9fcd-63d970d2b2ec.png">

### Acceptance Criteria

This is what done looks like:

* [ ] User is able to keyword search and return meaningful matches when searching within collections from collection landing pages

### Expected Behavior 

User is able to keyword search and return meaningful matches when searching within collections from collection landing pages.

### Actual Behavior

System returns meaningless results, i.e. every work in the collection.

**Comment by rmjaffe** [comment]
@rschwab @JessWaggs Can you please look into this?  I'm curious to get a better sense of what may be going on and what may be involved in correcting it.

**Comment by DiemBTran** [comment]
This passes Notch8 QA. See attached screen recording.

https://user-images.githubusercontent.com/29311858/144141117-b618ffc1-68c5-4a9a-b6eb-ceb73aa4b6cd.mp4

**Comment by rmjaffe** [comment]
This also looks good to me.  @rschwab @JessWaggs Anything wanting?

**Comment by rschwab** [comment]
@snehagunduraoUL was this in the mega pull request from N8? I'm confused about this being in the 'code review' column, maybe I'm missing something.

**Comment by snehagunduraoUL** [comment]
@rschwab No..not this one. The one I tested was #479

**Comment by sara-g** [comment]
@rschwab We have not sent this code yet, but if everything looks good we can send it today. Same with 479.

**Comment by rmjaffe** [comment]
@snehagunduraoUL Would it make best sense to leave this in staging and to not push to production until after we've descoobied?

**Comment by snehagunduraoUL** [comment]
Yeah..that makes sense unless this is a high priority to be on production.

**Comment by rmjaffe** [comment]
@snehagunduraoUL It is a top priority, but if it's a ton of effort or if there's a good risk of scooby-related issues (i.e., more bugs), we should wait.

**Comment by snehagunduraoUL** [comment]
@sara-g what do you think?
